signingscript/docker.d/init_worker.sh

```diff
@@ -28,16 +28,18 @@ export ZIPALIGN_PATH=/usr/bin/zipalign
 export PASSWORDS_PATH=$CONFIG_DIR/passwords.json
 export GPG_PUBKEY_PATH=$CONFIG_DIR/gpg_pubkey
 export WIDEVINE_CERT_PATH=$CONFIG_DIR/widevine.crt
-export AUTHENTICODE_TIMESTAMP_STYLE=null
+export AUTHENTICODE_TIMESTAMP_STYLE=old
 export AUTHENTICODE_CERT_PATH=/app/signingscript/src/signingscript/data/authenticode_dep.crt
 export AUTHENTICODE_CERT_PATH_202005=/app/signingscript/src/signingscript/data/authenticode_dep.crt
 export AUTHENTICODE_CA_PATH=/app/signingscript/src/signingscript/data/authenticode_dep_ca.crt
+export AUTHENTICODE_CA_TIMESTAMP_PATH=/usr/lib/ssl/certs/ca-certificates.crt
```

Comment:
I was going to say that we may need to populate https://github.com/mozilla-services/cloudops-infra/blob/master/projects/relengworker/k8s/charts/signing/templates/secret.yaml with this variable too but I don't think it's needed since it's local to this file and doesn't consume any public cloudops infra values nor the SOPS.